src/main/js/controls/Gantt/helpers/actionHelpers.js

```diff
@@ -276,8 +262,24 @@ const drawActionDataPoints = (scale, config, canvasSVG) =>
         .classed(styles.pointGroup, true)
         .each(function(dataPoint, index) {
             const dataPointSVG = d3.select(this);
-            renderSelectionPath(scale, config, dataPointSVG, dataPoint, index);
-            renderDataPointPath(scale, config, dataPointSVG, dataPoint, index);
+            if (dataPoint.y !== null) {
```

Comment:
There is nothing wrong with this approach but we could make use of `d3.select.filter` function instead?
It would work this way:
```js
d3.select("body")
    .selectAll("p")
    .data(
        [4, 8, 15, 16, 23, 42].filter(function(d) {
            return d < 10;
        })
    )
    .enter()
    .append("div")
    .text(function(d) {
        return d;
    });
```

Reply:
done here https://github.com/cerner/carbon-graphs/pull/164/commits/b4d0cd4a3fe85882be8c7acfd2c8cde97ca5dda0.